src/build

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+#
+# Compiles/caches and runs build.d with or without rdmd
```

Comment:
Why a Bash script?  Part of the motivation for using build.d is because not all platforms (specifically Windows) have Bash out of the box.  D works on all platforms, out of the box, by definition.

Reply:
this isn't meant for developers to use, only to bootstrap `build.d` from the makefile...since the CI builders don't seem to have `rdmd` installed

Reply:
If you have any better ideas let me know.  We could clone the tools repo and build rdmd if it is missing for example

Reply:
> only to bootstrap build.d from the makefile

Ok, I see.

> We could clone the tools repo and build rdmd if it is missing for example

That would be preferable to me.

You could also just have the makefile build `build.d` as a new target, yes?  This bash script has an awful lot of elaboration just to build one file.

Reply:
Ah yes, that would work better